tests/Functional/ContentPublicationRenderTest.php

```diff
@@ -276,6 +276,77 @@ public function testPublicationRendering(): void {
     $this->assertCount(4, $content_items);
     $this->assertContentHeader($content_items[3], 'Contact', 'contact');
     $this->assertContactEntityDefaultDisplay($content_items[3], 'publication_contact');
+
+    // Assert Organisation Contact.
+    $organisation_contact = $this->createContactEntity('organisation_contact', 'oe_general', 1);
+    $organisation_contact->save();
```

Comment:
Instead of this would be better to add `$contact->save()` in `ContentRenderTestBase::createContactEntity()`

Reply:
Our create* methods work inconsistent - createMedia*() save media and return saved entity

Reply:
I think you are right, I will add it to the createContactEntity to be consistent.